src/components/pages/user/list/download-logs/header.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, Select, Stack, Text } from "@chakra-ui/react";
+import { format } from "indian-number-format";
+import useTranslation from "next-translate/useTranslation";
+import React, { useMemo } from "react";
+
+import useDownloadLogsList from "../../common/use-download-log";
+
+export default function Header() {
+  const {
+    downloadLogData: { ag: aggregate },
+    addFilter,
+    resetFilter,
+    filter
+  } = useDownloadLogsList();
+  const { t } = useTranslation();
+
+  const totalCount: any = useMemo(() => {
+    return aggregate?.reduce((acc, item) => {
+      if (filter?.sourceType) {
+        return Object.values(
+          aggregate.find((item) => {
+            if (Object.keys(item)[0] === filter?.sourceType) {
+              return Object.keys(item)[0];
+            }
+          })
+        )[0];
+      } else {
+        return acc + Number(Object.values(item));
+      }
+    }, 0);
+  }, [filter]);
```

Comment:
I don't get this completely but probably better done at back-end you can follow observation/list page alike stats